taxonomies/categories.txt

```diff
@@ -18496,11 +18629,14 @@ wikidata:en:Q588378
 oqali_family:en: fr:Biscuits et gateaux industriels - Financiers
 
 <en:Biscuits
-en:Cat tongue
+en:Cat tongue, Cat's tongue cookie, Cat's tongue biscuit, Cigarette Russes, Russian cigarettes
```

Comment:
Cat's tongues and Russian cigarettes are two different kinds of biscuits, we should have separate categories for them.
It is ok to associate one ciqual category to multiple categories in the OFF taxonomy.

Reply:
OK, je me demandais justement si nous pouvions associer une catégorie CIQUAL à plusieurs catégories dans la taxonomie OFF... Merci pour l'info !

Reply:
Corrigé dans la prochaine PR